modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,500 @@
+
+*Type*: `string`
+
+```yml
+# Examples
+role: ACCOUNTADMIN
+```
+
+=== `database`
+
+The Snowflake database you want to write data to.
+
+*Type*: `string`
+
+=== `schema`
+
+The schema of the Snowflake database you want to write data to.
+
+*Type*: `string`
+
+
+=== `table`
+
+The Snowflake table you want to write data to.
+
+*Type*: `string`
+
+=== `private_key`
+
+The PEM-encoded private RSA key to use for authentication with Snowflake. You must specify a value for this field or the `private_key_file` field.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+=== `private_key_file`
+
+A `.p8`, PEM-encoded file to load the private RSA key from. You must specify a value for this field or the `private_key` field.
+
+
+*Type*: `string`
+
+
+=== `private_key_pass`
+
+If the RSA key is encrypted, specify the RSA key passphrase.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+=== `mapping`
+
+The xref:guides:bloblang/about.adoc[Bloblang `mapping`] to execute on each message.
+
+*Type*: `string`
+
+=== `init_statement`
+
+Optional SQL statements to execute immediately after this output connects to Snowflake for the first time. This is a useful way to initialize tables before processing data. 
+
+NOTE: Make sure any SQL statements are idempotent, so they do not cause issues when run multiple times after service restarts.
+
+*Type*: `string`
+
+```yml
+# Examples
+
+init_statement: |2
+  CREATE TABLE IF NOT EXISTS mytable (amount NUMBER);
+
+init_statement: |2
+  ALTER TABLE t1 ALTER COLUMN c1 DROP NOT NULL;
+  ALTER TABLE t1 ADD COLUMN a2 NUMBER;
+```
+
+=== `schema_evolution`
+
+Options to control schema updates as new columns are added to the pipeline.
+
+*Type*: `object`
+
+=== `schema_evolution.enabled`
+
+Whether schema evolution is enabled.
+
+*Type*: `bool`
+
+=== `schema_evolution.new_column_type_mapping`
+
+A mapping function from the Redpanda Connect data types to column data types in Snowflake. This mapping should result in the assignment of a string to the `root` variable with the data type for the new column in Snowflake.
+
+The input to this mapping is an object with the value and the name of the new column. For example: `{"value": 42.3, "name":"new_data_field"}"`.
+
+Overriding the mapping function, allows you to customize data types.
+
+*Type*: `string`
+
+*Default*: `"root = match this.value.type() {\n  this == \"string\" =\u003e \"STRING\"\n  this == \"bytes\" =\u003e \"BINARY\"\n  this == \"number\" =\u003e \"DOUBLE\"\n  this == \"bool\" =\u003e \"BOOLEAN\"\n  this == \"timestamp\" =\u003e \"TIMESTAMP\"\n  _ =\u003e \"VARIANT\"\n}"`
```

Comment:
Can we make this a code block instead of inline?

Reply:
Added a code block as requested.